ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/tap-aggregation.j2

```diff
@@ -0,0 +1,47 @@
+{% if tap_aggregation is arista.avd.defined and tap_aggregation.mode.exclusive.enabled is arista.avd.defined(true) %}
+
+## Tap Aggregation
+
+### Tap Aggregation Summary
+
+| Settings | Values |
+| -------- | ------ |
+| Mode Exclusive | {{ tap_aggregation.mode.exclusive.enabled }} |
+| Mode Exclusive Profile | {{ tap_aggregation.mode.exclusive.profile | arista.avd.default('tap-aggregation-default') }} |
```

Comment:
Add this entry only if defined. I am not sure if this is going to be default on all platforms. Even this could change in future.